direnvrc

```diff
@@ -3,6 +3,11 @@
 # Usage: nix_direnv_version <version_at_least>
 #
 # Checks that the nix-direnv version is at least as old as <version_at_least>.
+
+if ! has direnv_version || ! direnv_version 2.21.3; then
+  exit 1
```

Comment:
Cool. Could we get a nicer error message for this?

Reply:
Sure. Do you have wording you'd prefer?
`direnv_version` already prints to stderr. Would you like me to redirect that and print our own? Or would you prefer both the default message and a follow up to clarify it?

Reply:
Well, I took a stab while github was down. Please let me know what you think.

Reply:
Yep. that's better!

Reply:
Where is the `has` command coming from? The direnvrc fails to load for me in zsh with this change.

```
❯ zsh
has: command not found
nix-direnv: base direnv version is older than the required v2.21.3.
❯ direnv --version
2.30.3
❯ cat ~/.zshrc
[...]
eval "$(/nix/store/gx80b9p7xdyjmbkns86zl8kkkaz5bfsl-direnv-2.30.3/bin/direnv hook zsh)"
source /nix/store/s26xz2577x2a74zd460r84m7axzjz236-nix-direnv-1.6.0/share/nix-direnv/direnvrc
[...]
```

Reply:
`has`is documented in the direnv stdlib here: https://direnv.net/man/direnv-stdlib.1.html#codehas-ltcommandgtcode

I'm not sure why this isn't working for you. It might be worth opening an issue so that we can investigate this indepdently?

Reply:
I was holding it wrong. I included nix-direnv inside my zshrc instead of inside the direnvrc.

Reply:
Glad you got it sorted.